wire-tests/src/jvmTest/proto-kotlin/com/squareup/wire/protos/kotlin/Form.kt

```diff
@@ -215,6 +235,8 @@ data class Form(
       throw AssertionError()
     }
 
+    override fun equals(other: Any?): Boolean = other is ButtonElement
```

Comment:
Hmm, is this correct? What if the `unknownFields` are different? Implementation of `equals()` is copied from `JavaGenerator`.

Reply:
Looks broken!

Reply:
And probably should be fixed separately since it affects both generators